src/client.rs

```diff
@@ -1456,6 +1467,22 @@ impl LoginConfigHandler {
                 BoolOption::No
             })
             .into();
+        } else if name == "follow-remote-cursor" {
```

Comment:
If it's not in sciter's menu, don't send it for sciter version, it would be better to add these two menu,